tests/JWT.Tests.Common/SerializerTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using JWT.Serializers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace JWT.Tests
+{
+    [TestClass]
+    public class SerializerTests
+    {
+        [TestMethod]
+        public void Serializer_Should_Use_Correct_Default()
+        {
+            var serializer = JsonSerializerFactory.Serializer;
+            Console.WriteLine(serializer.GetType().Name);
```

Comment:
Please assert something. Or that's the test which always returns the same serializer?

Reply:
I have updated the test to test against the identified running dotnet version. As you can see all .net fw tests uses same version:
Dotnet version: .NETFramework,Version=v4.6.2

![image](https://user-images.githubusercontent.com/12826053/187000720-efd30327-14a6-4693-86dc-f46b0380ea5c.png)

I wanted to have a test where it was identified to be running .net46 for example and in that case return JsonNetSerializer as the serializer.